packages/core/src/lib/lib/useRenderer.ts

```diff
@@ -1,16 +1,34 @@
 import { onDestroy } from 'svelte'
 import { writable } from 'svelte/store'
 import {
+  ColorManagement,
+  LinearEncoding,
   PCFSoftShadowMap,
+  sRGBEncoding,
   WebGLRenderer,
-  type WebGLRendererParameters,
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  ColorManagement
+  type ColorSpace,
+  type TextureEncoding,
+  type WebGLRendererParameters
 } from 'three'
 import type { ThrelteContext } from '../types/types'
 import { watch } from './storeUtils'
 
+/**
+ * March 2023: Three.js is making a transition to a new color management system. Part of
+ * that is that the renderer will accept a `colorSpace` property rather than a
+ * property `encoding`. As a fallback for older three versions, we need to map
+ * the new `colorSpace` to the old `encoding`.
+ */
+const colorSpaceToEncoding: Record<ColorSpace, TextureEncoding> = {
+  srgb: sRGBEncoding,
+  'srgb-linear': LinearEncoding,
+  '': LinearEncoding
+}
+
+const rendererHasColorSpaceProperty = (renderer: any): renderer is { colorSpace: string } => {
+  return renderer.colorSpace !== undefined
```

Comment:
The plan is for this property to be named `renderer.outputColorSpace` rather than `renderer.colorSpace`, to differentiate the "working" color space used in the render pipeline from the "output" color space used when drawing to the canvas. 

Because threlte syntax would have this property on the `<Canvas />` component, including the "output" term felt redundant so I suggested omitting it there.